src/providers.ts

```diff
@@ -72,6 +74,8 @@ const providers: InternalProvider[] = [
   ["CIRRUS", "CIRRUS_CI"],
   ["CLOUDFLARE_PAGES", "CF_PAGES", { ci: true }],
   ["CLOUDFLARE_WORKERS", "WORKERS_CI", { ci: true }],
+  ["CLOUDRUN", "K_SERVICE"],
+  ["CLOUDRUN_JOB", "CLOUD_RUN_JOB"],
```

Comment:
For this i think we should add ci flag

Reply:
Anotherthing for clarity, maybe makes sense we call it `GOOGLE_CLOUDRUN`?

Reply:
yeah, probably makes sense. as it is in `AWS_AMPLIFY`  or `AZURE_...`